bijection-util/src/main/scala/com/twitter/bijection/twitter_util/UtilBijections.scala

```diff
@@ -78,6 +80,32 @@ trait UtilBijections {
     }
   }
 
+  private val futureConverter = {
+    val converter = new JavaFutureToTwitterFutureConverter()
+    converter.start()
+    converter
+  }
+
+  /**
+   * Injection from java futures to twitter futures
+   */
+  implicit def twitter2JavaFuture[A]: Bijection[TwitterFuture[A], JavaFuture[A]] = {
```

Comment:
yes, but having a thread started as a private val is kind of a non-starter, I think. It will impact serialization and it will consume resources for everyone who use this or not.